Cargo.toml

```diff
@@ -94,10 +106,17 @@ backend-llvm = [
     "wasmer-runtime/llvm",
     "wasmer-middleware-common-tests/llvm",
 ]
-backend-singlepass = [
-    "wasmer-singlepass-backend",
-    "wasmer-runtime/singlepass",
-    "wasmer-middleware-common-tests/singlepass",
+default-backend-singlepass = [
+    "backend-singlepass",
+    "wasmer-runtime/default-backend-singlepass",
+]
+default-backend-cranelift = [
+    "backend-cranelift",
+    "wasmer-runtime/default-backend-cranelift",
+]
+default-backend-llvm = [
```

Comment:
The default-backend features are intuitively less useful when using Wasmer as a CLI and in the abstract, I think it'd be better if we didn't expose these -- I'm sure you have a reason for doing it though.